### PR TITLE
Add a helpful description to the `filter` argument

### PR DESCRIPTION
**Pull request by danielbachhuber**
From #1884

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@WP-API/amigos #reviewmerge

File: lib/endpoints/class-wp-rest-posts-controller.php [hunk 1]
```diff
@@ -1595,7 +1595,9 @@ public function get_collection_params() {
 			'type'              => 'string',
 			'validate_callback' => array( $this, 'validate_user_can_query_private_statuses' ),
 		);
-		$params['filter'] = array();
+		$params['filter'] = array(
+			'description'       => 'Use WP Query arguments to modify the response; private query vars require appropriate authorization.',
+			);
```
Comment (rmccue) [inline lib/endpoints/class-wp-rest-posts-controller.php]: Extra indentation here, aka "the Daniel tab rule" ;)

Comment (danielbachhuber) [inline lib/endpoints/class-wp-rest-posts-controller.php]: a6264162aff34862365672e253be5f557b09bbfb

**Comment by rmccue** [comment]
Oh, should this be translated too? I _think_ so...

**Pull request merged**